Automattic/jetpack
# Social: Replace + icon with "Manage connections" button in editor

The + icon beside the connection list does more than adding a connection, thus we should replace that + icon with a button/link that says "Manage connections"

Related p1717676583920509-slack-C02JJ910CNL

## Proposed changes:

* Replace the + icon with connections management link/button

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






<details>
    <summary>To simulate a broken connection, you can make these changes (click to expand)</summary>

```diff
diff --git a/projects/packages/publicize/src/class-publicize.php b/projects/packages/publicize/src/class-publicize.php
index 9b52ce51f5..cb3234df0a 100644
--- a/projects/packages/publicize/src/class-publicize.php
+++ b/projects/packages/publicize/src/class-publicize.php
@@ -271,8 +271,10 @@ class Publicize extends Publicize_Base {
                $connection_results     = json_decode( wp_remote_retrieve_body( $response ), true );
                $connection_results_map = array();

+               $count = 0;
+
                foreach ( $connection_results as $connection_result ) {
-                       $connection_results_map[ $connection_result['connection_id'] ] = $connection_result['test_success'] ? 'ok' : 'broken';
+                       $connection_results_map[ $connection_result['connection_id'] ] = $connection_result['test_success'] && ( $count++ ) ? 'ok' : 'broken';
                }
                foreach ( $connections as $key => $connection ) {
                        if ( isset( $connection_results_map[ $connection['connection_id'] ] ) ) {
```
</details>

* Create a new post open Jetpack/Social sidebar
* Under "Share this post", confirm that you see the connections list
* Confirm that you see the "Manage connections" button below the connections list
* Click on it to confirm that it opens the connections modal
* Test the above for WPCOM sites
* Confirm that the "Manage connections" link points to Calypso

With connections
<img width="285" alt="image" src="https://github.com/Automattic/jetpack/assets/18226415/6c22504a-619e-4fa9-aa21-b69c53a4867d">
<img width="282" alt="image" src="https://github.com/Automattic/jetpack/assets/18226415/c027ed85-153c-49da-a92e-555bd76a8914">



Without connections
<img width="285" alt="image" src="https://github.com/Automattic/jetpack/assets/18226415/10c83d0d-f638-4eb8-98c0-31714f86cf4e">
<img width="290" alt="image" src="https://github.com/Automattic/jetpack/assets/18226415/b87f8fe2-30d4-4246-aae9-3fe080ab604b">
